.github/workflows/build.yml

```diff
@@ -92,23 +104,27 @@ jobs:
         run: poetry run poe checks
         working-directory: tests
 
-      - name: Install Rust ${{ env.rust_stable }}
+      - name: Install tools
+        run: |
+          sudo apt-get install -y cmake
+          cmake --version
+
+      - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ env.rust_stable }}
-          components: rustfmt, clippy
 
       - name: Build relay server
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: -p ya-relay-server --release
+          args: -p ya-relay-server
 
       - name: Build http client
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --example http_client --release
```

Comment:
Why build it in debug mode? Our tests depend on these binaries and we assert times there.
That means that when using debug builds in tests we test unreal setup and these asserts mean nothing.

Reply:
I removed `--release` because it takes time to compile it, with plan to move back to `--release` when Integration Tests job will be moved to self hosted runner and/or we will have tests that actually fail on debug build.

Reply:
I reverted both client and relay back to `--release` after recent Integration Tests failure.
The main problem are GH runners.